ts/components/FooterTopShadow.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
-import { View, StyleSheet } from "react-native";
-import { IOStyles } from "../../../../components/core/variables/IOStyles";
-import variables from "../../../../theme/variables";
+import { StyleSheet, View } from "react-native";
+import variables from "../theme/variables";
+import { IOStyles } from "./core/variables/IOStyles";
```

Comment:
Can we use the one exported from `@pagopa/io-app-design-system` instead?

Reply:
Sure! See https://github.com/pagopa/io-app/pull/5397/commits/76dab28696c414b8f07b1291bd5e432a7b1fb00f